packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/appearance/PaletteAppearanceSection.tsx

```diff
@@ -53,13 +56,19 @@ const useStyle = makeStyles()((theme) => ({
 }));
 
 export const PaletteAppearanceSection = ({
-  diagramElementId,
+  diagramElementIds,
   onBackToMainList,
 }: PaletteExtensionSectionComponentProps) => {
   const { classes } = useStyle();
   const { nodeLookup, edgeLookup } = useStoreApi<Node<NodeData>, Edge<EdgeData>>().getState();
-  const edge: Edge<EdgeData> | undefined = edgeLookup.get(diagramElementId);
-  const node: InternalNode<Node<NodeData>> | undefined = edge ? undefined : nodeLookup.get(diagramElementId);
+  const edgeElement: Edge<EdgeData> | undefined = edgeLookup.get(diagramElementIds[0] || '');
+  const nodeElement: InternalNode<Node<NodeData>> | undefined = edgeElement
+    ? undefined
+    : nodeLookup.get(diagramElementIds[0] || '');
+
+  if (diagramElementIds.length > 1) {
+    return null;
+  }
```

Comment:
Why not for now but end users want to be able to change the backend color of multiple nodes at once for example.